vere/reck.c

```diff
@@ -27,20 +27,18 @@ _reck_mole(u3_noun  fot,
            u3_noun  san,
            c3_d*    ato_d)
 {
-  u3_noun uco = u3do("slay", san);
-  u3_noun p_uco, q_uco, r_uco, s_uco;
+  u3_noun uco = u3dc("slaw", fot, san);
+  u3_noun p_uco, q_uco;
 
-  if ( (c3n == u3r_qual(uco, &p_uco, &q_uco, &r_uco, &s_uco)) ||
-       (0 != p_uco) ||
-       (0 != q_uco) ||
-       (c3n == u3r_sing(fot, r_uco)) )
+  if ( (c3n == u3r_cell(uco, &p_uco, &q_uco)) ||
+       (0 != p_uco) )
```

Comment:
While this is being changed, could you use `u3_nul` in place of `0`? (I know `0` vs `c3y` vs `u3_nul` is a long lost battle, but I'm drawn to such causes ...)

Reply:
Done.